<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open a new connection</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Open a new ssh connection</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>391</width>
     <height>117</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="hostnameLineEdit">
      <property name="placeholderText">
       <string>ssh-styled hostname</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="placeholderText">
       <string>optional</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="portLineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="placeholderText">
       <string> port (opt.)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="connectionNameLineEdit">
      <property name="placeholderText">
       <string>optional</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>connection name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>remote host</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>ssh keyfile</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>~/.ssh/id_rsa</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="saveOpenButton">
      <property name="text">
       <string>Save and Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openButton">
      <property name="text">
       <string>Open only</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>hostnameLineEdit</tabstop>
  <tabstop>portLineEdit</tabstop>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>connectionNameLineEdit</tabstop>
  <tabstop>saveOpenButton</tabstop>
  <tabstop>openButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
